src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/classes/undefined_et.cpp

```diff
@@ -135,6 +136,11 @@ fill_data(tensor->data<ov::element_type_traits<P>::value_type>(), S, L); break;
 }
 
 TEST_P(UndefinedEtSubgraphTest, CompareWithRefs) {
+    if (m_data_et == element::f16 && !ov::intel_cpu::hasHardwareSupport(m_data_et)) {
+        abs_threshold = 1.f;
+        rel_threshold = 0.1f;
+    }
+
```

Comment:
Usually we define the thresholds in the `SetUp` method

Reply:
Moved